assets/dev/js/editor/elements/collections/elements.js

```diff
@@ -13,7 +13,9 @@ var ElementsCollection = Backbone.Collection.extend( {
 		var ModelClass = Backbone.Model;
 
 		if ( attrs.elType ) {
-			ModelClass = elementor.hooks.applyFilters( 'element/model', ElementModel, attrs );
+			const registeredWidgetArgs = elementor.getRegisteredElementType( attrs.elType, attrs.widgetType );
+
+			ModelClass = elementor.hooks.applyFilters( 'element/model', registeredWidgetArgs?.Model ? registeredWidgetArgs.Model : ElementModel, attrs );
```

Comment:
// Todo: All elements should be registered and `registeredWidgetArgs.Model` must be exists